packages/zoe/src/zoeService/zoe.js

```diff
@@ -209,6 +209,34 @@ function makeZoe(vatAdminSvc) {
                 brandToPurse.init(brand, E(issuer).makeEmptyPurse());
               }
             }),
+        makeEmptySeat: (keyword, initialAllocation) => {
+          let currentAllocation = initialAllocation;
+
+          const { notifier, updater } = makeNotifierKit();
+
+          const seatAdmin = {
```

Comment:
I think we should use the same seatAdmin code and not have duplication here

Reply:
done